test/Schemes/AffineSchemes.jl

```diff
@@ -34,6 +69,21 @@
   mirr = SpecMor(Xstd, Xstd, [y, x, z])
   @test is_isomorphism(mirr)
   @test pullback(compose(inverse(mirr), mirr))(OO(Xstd)(x^2-34*z)) == OO(Xstd)(x^2-34*z+ f^2)
+  @test is_empty(EmptyScheme(QQ))
+  @test issubset(EmptyScheme(QQ),A3)
+  @test issubset(EmptyScheme(QQ),U)
+  @test !issubset(U,EmptyScheme(QQ))
+  @test issubset(X,A3)
+  @test !issubset(A3, X)
+  @test issubset(A3,A3)
+  @test issubset(intersect(A3,A3), A3)
+  @test dim(A3) == 3
+  @test dim(U) == 3
+  @test dim(V) == 3
```

Comment:
@simonbrandhorst This line throws an error (leading all tests to fail) because `V` is undefined